package-lock.json

```diff
@@ -149,7 +149,7 @@
         "webpack-cli": "5.1.4"
       },
       "engines": {
-        "node": ">=20.0.0 <24.0.0",
+        "node": ">=20.0.0 <22.0.0",
```

Comment:
is this intentional?

Reply:
no, it happened as a result of the temporary leo update

Reply:
(and because someone has modified `engines` in `package.json` but not `package-lock.json`)